app/helpers/repository_datatable_helper.rb

```diff
@@ -115,11 +115,14 @@ def prepare_simple_view_row_columns(repository_rows, repository, my_module, opti
     stock_consumption_permitted = has_stock_management && stock_consumption_permitted?(repository, my_module)
 
     repository_rows.map do |record|
+      record_info_url = unless record.repository.is_a?(RepositorySnapshot)
```

Comment:
unless record.repository.is_a?(RepositorySnapshot)
  row['recordInfoUrl'] = Rails.application.routes.url_helpers.repository_repository_row_path(record.repository, record)
end

put it before if reminders_enabled